diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/chapter6/EmptyPrimaryConstructorWarnTest.kt

```diff
@@ -0,0 +1,43 @@
+package org.cqfn.diktat.ruleset.chapter6
+
+import com.pinterest.ktlint.core.LintError
+import generated.WarningNames
+import org.cqfn.diktat.ruleset.constants.Warnings.EMPTY_PRIMARY_CONSTRUCTOR
+import org.cqfn.diktat.ruleset.rules.AvoidEmptyPrimaryConstructor
+import org.cqfn.diktat.ruleset.rules.DIKTAT_RULE_SET_ID
+import org.cqfn.diktat.util.LintTestBase
+import org.junit.jupiter.api.Tag
+import org.junit.jupiter.api.Test
+
+class EmptyPrimaryConstructorWarnTest: LintTestBase(::AvoidEmptyPrimaryConstructor) {
+    private val ruleId = "$DIKTAT_RULE_SET_ID:avoid-empty-primary-constructor"
+
+    @Test
+    @Tag(WarningNames.EMPTY_PRIMARY_CONSTRUCTOR)
+    fun `simple classes with empty primary constructor`() {
+        lintMethod(
+                """
+                    |class Some() {
+                    |   val a = 10
+                    |   constructor(a: String): this() {
+                    |       this.a = a
+                    |   }
+                    |}
+                    |
+                    |class Some1() {
+                    |   val a = 10
+                    |   companion object() {}
```

Comment:
```suggestion
                    |   companion object {}
```
companion object can't have parentheses